rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java

```diff
@@ -2672,6 +2681,92 @@ void voteFederationChange_add100Members_returnsSuccessResponseCodeAndFedSize100(
 
         }
 
+        @Test
+        void voteFederationChange_commit10MembersFederation_returnsSuccessResponseCodeAndPendingFedNullAndActiveFedIsTheVotedFed() {
+
+            // Arrange
+
+            Block executionBlock = mock(Block.class);
+
+            long creationBlockNumber = 1_000L;
+            when(executionBlock.getNumber())
+                .thenReturn(creationBlockNumber)
+                .thenReturn(federationMainnetConstants.getFederationActivationAge(activations) + creationBlockNumber);
```

Comment:
Why are these 2 chained? Returns first one and then the other?

Reply:
Refactored it to try to make it more clear.